dashboards/mongodb-prometheus/mongodb.json

```diff
@@ -0,0 +1,1714 @@
+{
+  "name": "MongoDB Prometheus",
+  "description": null,
+  "pages": [
+    {
+      "name": "Overview",
+      "description": null,
+      "widgets": [
+        {
+          "title": "",
+          "layout": {
+            "column": 1,
+            "row": 1,
+            "width": 4,
+            "height": 3
+          },
+          "visualization": {
+            "id": "viz.markdown"
+          },
+          "rawConfiguration": {
+            "text": "![MongoDB logo](https://upload.wikimedia.org/wikipedia/commons/thumb/9/93/MongoDB_Logo.svg/320px-MongoDB_Logo.svg.png)\n\nMongoDB is a highly scalable document-oriented NoSQL database.\n\nNew Relic's MongoDB integration collects data about the services running on your MongoDB infrastucture and sends it to our platform.\n\nUse this dashboard to get meaningful insights about instances, databases and collections and shards,\n\nRead more in the [docs](https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/mongodb-monitoring-integration/) (TBD)\n- - -\nIn this tab, the following variables filters data\n- Cluster"
```

Comment:
2 proposals.

1. ..... MongoDB integration collects data by using a Prometheus exporter about the services running on your MongoDB ....
2 ..... I understand the doc link once deployed is :  https://docs.newrelic.com/docs/infrastructure/host-integrations/host-integrations-list/mongodb/mongodb-monitoring-integration-new/

Reply:
Done